### PR TITLE
Dtspo 12577 new agent

### PR DESCRIPTION
**Pull request by viktoriyabogdanova**
### JIRA link (if applicable) ###
DTSPO-12577

### Change description ###
Change to 1 cluster only

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: apps/azure-devops/sbox/base/kustomization.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@ resources:
   - azure-devops-token.yaml
   - ../../azure-devops-agent-keda/azure-devops-agent.yaml
 patchesStrategicMerge:
-  - ../../azure-devops-agent-keda/ptlsbox.yaml
\ No newline at end of file
+  - ../../azure-devops-agent-keda/sbox.yaml
```
Comment (thomast1906) [inline apps/azure-devops/sbox/base/kustomization.yaml:9]: `../../azure-devops-agent-keda` references need moved to just one cluster, this case just 00

**Review by thomast1906 — CHANGES_REQUESTED** [review]

File: apps/azure-devops/sbox/00/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: azure-devops
```
Comment (thomast1906) [inline apps/azure-devops/sbox/00/kustomization.yaml:3]: remove duplicate value `namespace`

**Review by thomast1906 — APPROVED** [review]

File: apps/azure-devops/sbox/01/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../base
+  - ../../azure-devops-agent/azure-devops-agent.yaml
```
Comment (adusumillipraveen) [inline apps/azure-devops/sbox/01/kustomization.yaml:5]: You will have to remove old one on second cluster for ur test to work.

File: apps/azure-devops/sbox/00/kustomization.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
```
Comment (thomast1906) [inline apps/azure-devops/sbox/00/kustomization.yaml:3]: missing `- ../base`

**Pull request merged**